packages/terra-divider/src/terra-dev-site/doc/example/DividerWithText.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved, import/extensions
+import Divider from 'terra-divider/lib/Divider';
+import DividerExampleTemplate from './DividerExampleTemplate';
+import Moment from '../../../../../../node_modules/moment/moment';
```

Comment:
Let's avoid pulling in moment here. My preference would be we just use a string for this demo.

e.g. `<Divider text="01/01/2019" />`

Reply:
Thought that might have been pushing it - changed! [81179c5](https://github.com/cerner/terra-core/pull/2296/commits/81179c5398cb4eeb8c820f4f76ec2d946aaea09b)